docs/event-schemas/srm/SequenceRunStateChange.json

```diff
@@ -56,48 +54,44 @@
         "type": "object",
         "required": [
           "id",
-          "instrument_run_id"
+          "instrumentRunId",
+          "runVolumeName",
+          "runFolderPath",
+          "runDataUri",
+          "sampleSheetName",
+          "startTime",
+          "endTime",
```

Comment:
You are keeping one record for each SequenceRun, right?
(i.e. there are defined start/end times and no intermediate state-bound times)

Reply:
Yes, that right. One record for each run; unique by `instrumentRunId`.